libs/full/performance_counters/src/counters.cpp

```diff
@@ -506,7 +506,7 @@ namespace hpx { namespace performance_counters {
     {
         if (type < counter_text || type > counter_raw_values)
```

Comment:
@hkaiser Shouldn't the unscoped `counter_raw_values` here provide a deprecation warning? I don't see any when I run locally.

Reply:
You'd have to touch on all code that uses the enumeration values. Otherwise, the CI builders will fail (as soon as we bump the version to V1.9).

Reply:
Great! Done on my last commit. Let's resolve and wait for the tests.